tests/cursor/bug1713-001.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+MongoDB\Driver\Cursor PHPC-171: The cursor current method does not return anything
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+<?php skip_if_not_clean(); ?>
```

Comment:
Technically, I don't think this matters since we don't actually care if the collection is dropped. Whether it exists or not, we're just testing that `php_phongo_cursor_t.visitor_data.zchild` is zeroed out before the first call to `rewind()` or `next()`.

That said, no objections to keeping this in place.

Reply:
Noted. I've dropped the call, no reason to clear the database if we don't care about its contents.